src/Command/Branch/BranchSyncCommand.php

```diff
@@ -28,18 +30,69 @@ protected function configure()
     {
         $this
             ->setName('branch:sync')
-            ->setDescription('Syncs local branch with its upstream version')
-            ->addArgument('branch_name', InputArgument::OPTIONAL, 'Branch name to sync')
+            ->setDescription('Synchronises a local branch with a remote branch')
+            ->addArgument('source_branch', InputArgument::OPTIONAL, 'Remote branch-name to use as (new) base')
+            ->addArgument('source_remote', InputArgument::OPTIONAL, 'Remote to pull from (defaults to [your-username])')
             ->addArgument(
-                'remote',
+                'dest_remote',
```

Comment:
The arguments are never executed with a name ;)
Options are, and left these unchanged (as much as possible).

So using `branch:sync sstok master` will still work :+1: 

One thing that is a BC break, is that instead of forcing the local branch to equal the remote
branch it uses rebase by default. But then again using rebase is saver then using `git reset` by default.